trello/view_model.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+
+class ViewModel:
+
+    def __init__(self, items):
+        self._items = items
+
+    @property
+    def items(self):
+        return self._items
+
+    def get_all_item_of_type(self, list_name):
+        item_list = []
+        for item in self.items:
+            if item.status == list_name:
+                item_list.append(item)
+        return item_list
+
+    def show_all_done_items(self): 
```

Comment:
Having these as methods is fine, but an alternative would be to make them also properties, and then we can access them like `view_model.done_items`